package-lock.json

```diff
@@ -20634,7 +20634,7 @@
 				},
 				"node-pre-gyp": {
 					"version": "0.12.0",
-					"resolved": "https://registry.npmjs.org/node-pre-gyp/-/node-pre-gyp-0.12.0.tgz",
+					"resolved": false,
```

Comment:
I neglected to catch this in review, but these changes aren't expected, and are causing local changes to be introduced to the branch when running `npm install` on `master`.

See https://github.com/WordPress/gutenberg/issues/16157#issuecomment-601187274

In trying to debug this, could you check to see which version of NPM you're using?

See also my proposed (partial) solution: #21306

Reply:
Fixed (reverted) in c7a3c2ca17f6e4572f4570ff45f9b4afded5ebe7

Reply:
Whoops, thanks a lot! Not sure at which point I introduced this.

Using npm 6.14.4 if that helps.

Reply:
>Using npm 6.14.4 if that helps.

It does help to know. It's contrary to what I was hoping, however 😬 Since my conclusions from https://github.com/WordPress/gutenberg/issues/16157#issuecomment-601187274 and #21306 were based on assumptions that it's an issue with older versions of NPM. But 6.14.4 is the latest (at this point in time).

Reply:
I just updated it the other day, so there's a chance I introduced the changes here before I updated to that version.